### PR TITLE
Add Status badges

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Preview:

image

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # NVIDIA device plugin for Kubernetes
 
+[![End-to-end Tests](https://github.com/NVIDIA/k8s-device-plugin/actions/workflows/e2e.yaml/badge.svg)](https://github.com/NVIDIA/k8s-device-plugin/actions/workflows/e2e.yaml) [![Go Report Card](https://goreportcard.com/badge/github.com/NVIDIA/k8s-device-plugin)](https://goreportcard.com/report/github.com/NVIDIA/k8s-device-plugin) [![Latest Release](https://img.shields.io/github/v/release/NVIDIA/k8s-device-plugin?include_prereleases)](https://github.com/NVIDIA/k8s-device-plugin/releases/latest)
```
Comment (elezar) [inline README.md:3]: Do we want `include_prereleases` here?

**Review by elezar — COMMENTED** [review]
Looks good! Minor comment on prerelease versions.

**Review by elezar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ArangoGutierrez) [reply README.md:3]: up to us. I am not against or in favor

Comment (ArangoGutierrez) [reply README.md]: removed

**Pull request merged**